Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'rubocop'
```

Comment:
`[Optional]` You should add `rspec` to your gemfile so it gets installed on running `bundle install`.

Reply:
Added